Search/Adapter/ElasticSearchAdapter.php

```diff
@@ -68,8 +68,6 @@ public function __construct(
      */
     public function index(Document $document, $indexName)
     {
-        $indexName = $this->localizationStrategy->localizeIndexName($indexName, $document->getLocale());
```

Comment:
All the localization has been moved "up" to the SearchManager. The adapters are now localization agnostic.